rts/lra/narayana-lra/src/main/java/io/narayana/lra/provider/ParticipantStatusOctetStreamProvider.java

```diff
@@ -0,0 +1,33 @@
+package io.narayana.lra.provider;
+
+import org.eclipse.microprofile.lra.annotation.ParticipantStatus;
+
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.MessageBodyWriter;
+import javax.ws.rs.ext.Provider;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+@Provider
+@Produces(MediaType.APPLICATION_OCTET_STREAM)
```

Comment:
Since the actual body that the writer is writing is a java String (enum#name()) why did you choose an octet stream in preference to MediaType.TEXT_PLAIN?

Reply:
This is exactly the point of this issue. application/octet-stream is a default media type in RESTEasy, so this provider allows to declare JAX-RS Complete, Compensate, Status, etc. without `@Produces()` annotation.